### PR TITLE
fixed error handling for mysql/mariadb

### PR DESCRIPTION
**Pull request by sasank-chaganty**
* MySQL errors now display on the frontend
* added a `get_error_message()` function to ServerConnections, so that each driver is able to output an error message from when it experiences a DatabaseError, independently of other drivers.

Capture

### CODE REVIEW

File: ossdbtoolsservice/driver/types/psycopg_driver.py [hunk 1]
```diff
@@ -246,6 +246,12 @@ def get_database_size(self, dbname: str):
         """
         pass
 
+    def get_error_message(self, error) -> str:
+        """
+        Get the message from DatabaseError instance 
+        """
+        return "ERROR: " + error.diag.message_primary
```
Comment (KevinMicrosoft) [inline ossdbtoolsservice/driver/types/psycopg_driver.py:253]: Why do we need to add the "ERROR" string here? This now throws an error in our tests because we throw empty errors in our tests where error.diag.message_primary is None.

Comment (sasank-chaganty) [reply ossdbtoolsservice/driver/types/psycopg_driver.py:253]: Good question, my reasons for that are: (1) I was trying to make it look more like the error that shows up in PgAdmin, which I think users might be more used to seeing (2) Makes it more clear that it is an error and not just a warning
match pgadmin

I'm looking into how to use the `pgerror` property from each error [(Documented here)](https://www.psycopg.org/docs/module.html#exceptions) since that gives a much better error with line numbers and an arrow pointing out where exactly the error occurred.
